### DISCUSSION TITLE
How to count total updated or inserted rows

### DISCUSSION THREAD

**Discussion by anhthang** [supabase/supabase#9575] [Questions]
Hi there,

I'm using supabase-js with upsert function and I want to know how many rows was updated or new rows inserted.

Does anyone know how to do it?

**Comment by steve-chavez** [comment]
Hello,

This is possible by using the `xmax` column as described here https://github.com/supabase/postgrest-js/issues/143

**Reply by anhthang** [reply]
Do you have any sample that working for you? I have tried few ones but it seems not working for me
